app/src/main/res/menu/episodes.xml

```diff
@@ -30,4 +30,39 @@
         android:title="@string/favorite_episodes_label"
         custom:showAsAction="always"/>
 
+    <item
```

Comment:
Instead of copying the sort menu items here, please re-use them, like it is done in #6210.